### DISCUSSION TITLE
required torch 2.1.2

### DISCUSSION THREAD

**Discussion by 832r329rvuj20r2** [TheLastBen/fast-stable-diffusion#2758] [Q&A] [answered]
You are running torch 2.1.0+cu121.
The program is tested to work with torch 2.1.2.
To reinstall the desired version, run with commandline flag --reinstall-torch.
Beware that this will cause a lot of large files to be downloaded, as well as
there are reports of issues with training tab on the latest version.

Use --skip-version-check commandline argument to disable this check.

Thank you for always ! needs updating note

**Comment by nu9886** [comment]
i have problem this.   same same same

**Comment by TheLastBen** [comment, marked as answer]
does this prevent A1111 from working?

**Reply by nu9886** [reply]
Yes, I have 2 actions that I'm not sure are related to the problem.
- I have more than 100 units on Collab, but I haven't paid for them in over 1 month.
- I deleted the stabledeffusion files in my drive, but not the files ending with webui.

**Reply by 832r329rvuj20r2** [reply]
new note is working !
thank you kind ben
